core/src/epicli/data/common/ansible/playbooks/roles/haproxy_runc/tasks/install-packages-Debian.yml

```diff
@@ -0,0 +1,7 @@
+---
+- name: Install containerd.io package for Debian family
+  apt:
```

Comment:
Is it the correct place to install packages? Should it be in 'repository' role?

Reply:
So the "repository" is reponsible for downloading images, files etc (handles offline case as well). It hosts them in apache and registry services - so later each component / role can just use apt or yum do install them in the OS. Having said that I think the answer is yes, this looks to me as, a correct place to do it. 🤔 What do you suggest instead?

Reply:
Ok, now this is clear.